### PR TITLE
accordion default classname

### PR DESCRIPTION
**Pull request by Lendemor**
```
rx.accordion.root(
    rx.accordion.item(
        rx.accordion.header(
            rx.accordion.trigger(
                "Item 1",
                rx.accordion.icon(),
            ),
        ),
        rx.accordion.content(
            rx.text("Item 1"),
        ),
    ),
    rx.accordion.item(
        header="Item 2",
        content=rx.text("Item 2"),
    ),
)
```

### CODE REVIEW

**Review by picklelo — APPROVED** [review]

**Review by masenf — COMMENTED** [review]
Can we get the same treatment for AccordionHeader and AccordionItem

Also, can you move the definition of `slideUp` and `slideDown` to AccordionRoot

**Comment by Lendemor** [comment]
> Can we get the same treatment for AccordionHeader and AccordionItem
> 
> Also, can you move the definition of `slideUp` and `slideDown` to AccordionRoot

can do

File: reflex/components/radix/primitives/accordion.py [hunk 1]
```diff
@@ -493,6 +515,11 @@ def create(
...
         value = props.pop("value", get_unique_variable_name())
 
+        if "AccordionItem" not in (
+            cls_name := props.pop("class_name", "AccordionItem")
+        ):
+            cls_name = f"{cls_name} AccordionItem"
+
         if (header is not None) and (content is not None):
             children = [
                 AccordionHeader.create(
```
Comment (masenf) [inline reflex/components/radix/primitives/accordion.py:525]: we can simplify the following code now: removing the `class_name` specification and replacing `Icon` with the new `AccordionIcon`.

people looking to use the low level API will probably copy this as an example

**Review by masenf — APPROVED** [review]

File: reflex/components/radix/primitives/accordion.py (hunk 1 above)
Comment (Lendemor) [reply reflex/components/radix/primitives/accordion.py:525]: sure let's do that

**Pull request merged**